frontend/src/pages/DashboardPage/ControlButtons/ControlButtons.jsx

```diff
@@ -175,6 +181,72 @@ const ControlButtons = ({ onDownload }) => {
         >
           Download Graph
         </button>
+
+        {/* Bootstrap Modal for model upload instructions */}
+        {showModal && (
+            <div className="modal show" style={{ display: "block" }}>
+              <div className="modal-dialog">
+                <div className="modal-content">
+                  <div className="modal-header">
+                    <h5 className="modal-title">Model Upload Instructions</h5>
+                    <button
+                        type="button"
+                        className="close"
+                        style={{
+                          backgroundColor: "#45a049",
+                          borderColor: "#45a049",
+                          fontSize: "0.875rem",  // Smaller font size
+                          padding: "0.25rem 0.5rem",  // Smaller padding
+                          borderRadius: "0.2rem"  // Optional: round corners slightly
+                        }}
+                        onClick={closeModal}
+                    >
+                      &times;
+                    </button>
+                  </div>
+                  <div className="modal-body">
+                    <p><strong>File format:</strong> The file must be in <code>.pkl</code> format.</p>
+                    <p><strong>File size:</strong> The file size must be less than 1 MB.</p>
```

Comment:
Are we checking in backend if file is more than 1 MB?

Reply:
We haven't added any such conditions yet.